3rdparty/python/pytest.lock

```diff
@@ -4,7 +4,7 @@
 //
 // --- BEGIN PANTS LOCKFILE METADATA: DO NOT EDIT OR REMOVE ---
 // {
-//   "version": 2,
+//   "version": 3,
```

Comment:
Not super safe because I'm actively changing what v3 means. You can manually switch to v2 by changing this line to be `2`, and then remove the `requirement_constraints: []` line

Reply:
👍 will update this one and the other changed lockfile